diktat-rules/src/test/resources/test/funcTest/FunctionalTestFile.kt

```diff
@@ -3,3 +3,13 @@ package test.funcTest
 class FunctionalTestFile {
     val AAAA = 5
 }
+
+fun `method name incorrect, part 4`() {
```

Comment:
Please remove it from TestFile, you should add it into smoke test to test this case on the whole rule set provdier.